zerver/lib/create_user.py

```diff
@@ -123,5 +123,8 @@ def create_user(email: str, password: Optional[str], realm: Realm,
 
     recipient = Recipient.objects.create(type_id=user_profile.id,
                                          type=Recipient.PERSONAL)
+    user_profile.personal_recipient = recipient
+    user_profile.save()
```

Comment:
Should we add an `update_fields` here?  Would save some network transfer.

Reply:
good point, fixed!